.github/actions/rust-sccache/action.yml

```diff
@@ -0,0 +1,71 @@
+name: rust sccache
```

Comment:
we should look into moving this to a shared repo eventually (cc @galargh)

Reply:
Agree 👍

Reply:
That's a great idea. I'm going to track progress on it here: https://github.com/protocol/w3dt-stewards/issues/38